resources/user-agents/apps/microsoft/microsoft-office.json

```diff
@@ -16,13 +16,8 @@
         "Browser_Maker": "Microsoft Corporation",
```

Comment:
Why isn't it merge with 
> browscap/resources/user-agents/apps/microsoft/microsoft-office-2013.json

Reply:
There are 6 files for MS Office. This one file may be optimized.